### ISSUE TITLE
Bring back state level data for US

### ISSUE THREAD

**Issue by josephsdavid** [CSSEGISandData/COVID-19#1468]
Please

**Comment by josephsdavid** [comment]
Deprecating that in the time series section is going to break a ***ton*** of stuff

**Comment by MKastuv** [comment]
Please bring back the statewise data.

**Comment by nickblink** [comment]
yes agreed my project is using that state level data - it would be really appreciated!

**Comment by rtroha** [comment]
I assume state level data can be incorporated into the US specific file that will also have county data.  They need to add the state as a column, or the FIPS code.  You could always sum the total of cases of all counties in a given state to back into that data if need be

**Comment by josephsdavid** [comment]
> I assume state level data can be incorporated into the US specific file that will also have county data. They need to add the state as a column, or the FIPS code. You could always sum the total of cases of all counties in a given state to back into that data if need be

State is a column already in the time series data, however changing and deprecating it abruptly has likely broken the code of thousands of people. Also, the county level data in the time series (which is what I am referring to) does not add up to the state level data. For example, in Dallas, which has had deaths for DAYS, there are zero deaths in the county level time series data...

**Comment by mtempler** [comment]
US States, please.  We can get more information about China and each of its provinces than we can about the US.

County as soon as the source commits to some level of data quality .  Please.

**Comment by plorch** [comment]
Losing the ability to see trends in US states is a big setback.  There are tons of ways to see country wide, state/province, and even county wide totals, but the trend allowed us to see how our jurisdiction in the state compared to others.  Since testing is still rare, death rates derived from trend-lines is crucial.

**Comment by auf-rd** [comment]
they just don't care :) more data is more info. You need to adapt guys.
Can you share your models/sites ? (ads free)
[Here is mine](https://covid-19-stats.info)

**Comment by covid-19-stats** [comment]
They just don't care guys. More data is more information. You have to adapt to the "giant".
Can you share you models/apps ?
[Here is mine](https://covid-19-stats.info)

**Comment by mtempler** [comment]
The time-series data for states and counties can be generated from the daily reports.  

[Decision Machine COVID-19 Geiger Counter](https://www.decision-machine.com/covid19)

**Comment by josephsdavid** [comment]
> They just don't care guys. More data is more information. You have to adapt to the "giant".
> Can you share you models/apps ?

There is no way to adapt—the information we need is missing

**Comment by josephsdavid** [comment]
> The time-series data for states and counties can be generated from the daily reports.
> 
> [Decision Machine COVID-19 Geiger Counter](https://www.decision-machine.com/covid19)

Be careful, the county level data is wrong
